label_sync/labels.yaml

```diff
@@ -626,7 +626,19 @@ default:
       addedBy: prow
     - color: ffaa00
       description: Denotes a PR that should be squashed by tide when it merges.
-      name: tide/squash
+      name: tide/merge-method-squash
+      target: prs
+      addedBy: humans
+      previously:
+        - name: tide/squash
+    - color: ffaa00
+      description: Denotes a PR that should be rebased by tide when it merges.
+      name: tide/merge-method-rebase
+      target: prs
+      addedBy: humans
+    - color: ffaa00
+      description: Denotes a PR that should use a standard merge by tide when it merges.
+      name: tide/merge-method-merge
```

Comment:
`merge-method-merge` seems strange to me... why not:
- `tide/squash`
- `tide/rebase`
- `tide/merge`

Reply:
Might be nice to keep existing behavior, so people aren't confused when they issue `/label tide/squash`

Reply:
That was the original idea but I thought the last one, `tide/merge`, would be a little confusing. I could see someone misunderstand its purpose and thinking it's related to somehow asking tide to take action on the merge vs having something to do with the method of the merge once it's performed.